node/network/statement-distribution/src/vstaging/grid.rs

```diff
@@ -855,18 +863,20 @@ impl StatementFilter {
 	}
 }
 
+#[derive(Debug, Clone)]
 struct MutualKnowledge {
-	// Knowledge they have about the candidate. `Some` only if they
+	// Knowledge the remote peers have about the candidate. `Some` only if they
```

Comment:
I actually prefer "them" over "the remote peers" because we have a `MutualKnowledge` instance for each relevant peer. "the peer" would be better; some docs on `MutualKnowledge` as a struct wouldn't hurt either!

Reply:
Ah, "they" had me thinking in terms of plural, but "the peers" is indeed confusing. Appreciate the feedback!